server/src/main/java/org/apache/cassandra/sidecar/cdc/CdcConfigImpl.java

```diff
@@ -333,4 +334,29 @@ public ScheduleDecision scheduleDecision()
             return shouldSkip ? ScheduleDecision.SKIP : ScheduleDecision.EXECUTE;
         }
     }
+
+    enum ConfigKeys
+    {
+        DC("dc"),
+        LOG_ONLY("log_only"),
+        PERSIST_STATE("persist_state"),
+        ENV("env"),
+        KAFKA_TOPIC("topic"),
+        KAFKA_TOPIC_FORMAT_TYPE("topic_format_type"),
+        CDC_ENABLED("cdc_enabled"),
+        JOB_ID("jobId"),
+        WATERMARK_SECONDS("watermark_seconds"),
+        MICRO_BATCH_DELAY_IN_MILLIS("microbatch_delay_millis"),
+        MAX_COMMIT_LOGS("max_commit_logs"),
+        MAX_WATERMARKER_SIZE("max_watermarker_size"),
+        FAIL_KAFKA_ERRORS("fail_kafka_errors"),
+        FAIL_KAFKA_TOO_LARGE_ERRORS("fail_kafka_too_large_errors"),
+        PERSIST_DELAY_MILLIS("persist_delay_millis");
+        private final String name;
+
+        ConfigKeys(String name)
+        {
+            this.name = name;
+        }
+    }
```

Comment:
There is no need to repeat the string..

```suggestion
    enum ConfigKeys
    {
        DC,
        LOG_ONLY,
        PERSIST_STATE,
        ENV,
        KAFKA_TOPIC,
        KAFKA_TOPIC_FORMAT_TYPE,
        CDC_ENABLED,
        JOB_ID,
        WATERMARK_SECONDS,
        MICRO_BATCH_DELAY_IN_MILLIS,
        MAX_COMMIT_LOGS,
        MAX_WATERMARKER_SIZE,
        FAIL_KAFKA_ERRORS,
        FAIL_KAFKA_TOO_LARGE_ERRORS,
        PERSIST_DELAY_MILLIS;

        private final String lowcaseName;

        ConfigKeys()
        {
            this.lowcaseName = name().toLowerCase(Locale.ENGLISH);
        }
    }
```